js/controllers/newsliController.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+(function (exports) {
+  exports.NewsliController = function (ArticleList, ArticleListView, ArticleView) {
+    articleList = new ArticleList;
+    articleListView = new ArticleListView(articleList)
+    articleView = new ArticleView;
```

Comment:
I would generally recommend instantiation these and then passing in the instances, rather than passing in the classes and then instantiating. The latter pattern is generally only used when you need to generate instances at run time e.g. you have received some information from an API and you want to create a model with that information.